docs/process-typed.md

```diff
@@ -24,9 +24,14 @@ process hello {
 }
 ```
 
-In order to use this feature, you'll need to:
+To use this feature:
+
+1. Set the `NXF_SYNTAX_PARSER` environment variable to `v2`:
```

Comment:
I would still phrase it as "Enable the strict syntax by setting..." so that there is a link to the strict syntax. Enabling this flag in practice will require other changes to pipeline code which are explained on that page